src/librustc_error_codes/error_codes/E0689.md

```diff
@@ -1,13 +1,16 @@
-This error indicates that the numeric value for the method being passed exists
-but the type of the numeric value or binding could not be identified.
+A method was called on an ambiguous numeric type.
 
-The error happens on numeric literals:
+Erroneous code example:
 
 ```compile_fail,E0689
-2.0.neg();
+2.0.neg(); // error!
 ```
 
-and on numeric bindings without an identified concrete type:
+This error indicates that the numeric value for the method being passed exists
+but the type of the numeric value or binding could not be identified.
```

Comment:
I believe this would be best arrange above the example like the below description which `this` refers to. Or maybe it would be better if we use `This error` rather than `The error` for the text below to avoid confusion.

Reply:
Thanks for the feedback. Most of the error messages are of th esimilar fashion and I think it is fine for now.